SmartDeviceLinkTests/ProxySpecs/SDLSecondaryTransportManagerSpec.m

```diff
@@ -552,11 +580,34 @@ + (void)swapConnectionMethods {
                     testTransportEventUpdateMessage = [[SDLV2ProtocolMessage alloc] initWithHeader:testTransportEventUpdateHeader andPayload:testTransportEventUpdatePayload.data];
                 });
 
-                it(@"should transition to Connecting state", ^{
+                it(@"should transition to Connecting state but before RAIR is sent", ^{
+                    dispatch_sync(testStateMachineQueue, ^{
+                        [manager.stateMachine transitionToState:SDLSecondaryTransportStateConfigured];
+                    });
+
+                    [testPrimaryProtocol handleBytesFromTransport:testTransportEventUpdateMessage.data];
+                    [NSThread sleepForTimeInterval:0.1];
+
+                    expect(manager.stateMachine.currentState).to(equal(SDLSecondaryTransportStateConnecting));
+                    expect(manager.secondaryProtocol.securityManager).to(beNil());
+                    OCMVerifyAll(testStreamingProtocolDelegate);
+                });
+
+                it(@"should transition to Connecting state after RAIR is sent", ^{
```

Comment:
Again poor documentation / naming. Is this "it should transition to the connecting state after the library becomes ready when a security manager is set"?